.eslintignore

```diff
@@ -0,0 +1,6 @@
+node_modules
+bin
+gatsby-config.js
+gatsby-node.js
+.eslintrc.js
+commitlint.config.js
```

Comment:
I had to add these here to stop getting errors like this:
```
/Users/brian/Websites/4K/emulsify-website/.eslintrc.js
  0:0  error  Parsing error: "parserOptions.project" has been set for @typescript-eslint/parser.
The file does not match your project config: .eslintrc.js.
The file must be included in at least one of the projects provided
```

It didn't seem like I should need to add them here though, so if you know how to properly configure this, please let me know!

Reply:
This error is being thrown because in your `eslint.tsconfig.json` , you're instructing Typescript to ignore js files outside of the `src` directory. 

Instead of ignoring all js files in the root, I'd recommend opening up that `src/**/*.js` include so it doesn't exclude files in the root: `**/*.js`, I think is what you're looking for, or perhaps even `./*.js`. (here's an example: https://github.com/emulsify-ds/emulsify-cli/blob/develop/tsconfig.eslint.json)

Reply:
That makes sense now. Thanks!